src/scripts/Blacklist.elm

```diff
@@ -0,0 +1,12 @@
+module Blacklist exposing (contains, length)
+
+
+length : Int
```

Comment:
Where is this needed?

Reply:
Artifact from an old architecture - safe to delete!